src/frontend/web_application/src/scenes/Account/components/AccountOpenPGPKeys/index.js

```diff
@@ -0,0 +1,50 @@
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+import Presenter from './presenter';
+import * as openPGPKeychain from '../../../../store/modules/openpgp-keychain';
+
+const privateKeysSelector = createSelector(
+  state => state.openPGPKeychain,
+  openPGPKeychainState => openPGPKeychainState.privateKeys
+      .map(fingerprint => openPGPKeychainState.keychainByFingerprint[fingerprint]),
+);
+
+const importFormSelector = createSelector(
+  state => state.openPGPKeychain,
+  openPGPKeychainState => openPGPKeychainState.importForm,
+);
+const isLoadingSelector = createSelector(
+  state => state.openPGPKeychain,
+  openPGPKeychainState => openPGPKeychainState.isLoading,
+);
+
+const mapStateToProps = state => ({
+  privateKeys: privateKeysSelector(state),
+  importForm: importFormSelector(state),
+  isLoading: isLoadingSelector(state),
+});
+
+const mapDispatchToProps = dispatch => ({
+  onDeleteKey: ({ fingerprint }) => {
+    dispatch(openPGPKeychain.deleteKey(fingerprint));
+  },
+  onImportKey: ({ publicKeyArmored, privateKeyArmored }) => {
+    dispatch(openPGPKeychain.importKeyPairChains(publicKeyArmored, privateKeyArmored));
+  },
+  onGenerateKey: ({ name, email, passphrase }) => {
+    dispatch(openPGPKeychain.generate(name, email, passphrase));
```

Comment:
You could keep the same signature as your action-creators', their return value, reducing boilerplate code and avoiding typos by using redux' `bindActionCreators` https://github.com/reactjs/redux/blob/master/src/bindActionCreators.js

```js
import { bindActionCreators } from 'redux';

const mapDispatchToProps = dispatch => ({
  onDeleteKey: openPGPKeychain.deleteKey,
  ...
});
```